modules/clickhouse/clickhouse_test.go

```diff
@@ -210,13 +210,29 @@ func TestClickHouseWithConfigFile(t *testing.T) {
 			defer conn.Close()
 
 			// perform assertions
-			data, err := performCRUD(conn)
+			data, err := performCRUDWithRetry(conn)
 			assert.NoError(t, err)
 			assert.Len(t, data, 1)
 		})
 	}
 }
 
+func performCRUDWithRetry(conn driver.Conn) (tt []Test, err error) {
```

Comment:
Could we avoid named return parameters? There is no specific contributing rule for it, although I do prefer not using them if possible (we do have it in a few places, planned to refactor it at some point)

Reply:
I'd also merge both methods into just one, as we are not going to use `performCRUD` by itself.

Reply:
> Could we avoid named return parameters?

ok, i will update the method

> I'd also merge both methods into just one, as we are not going to use performCRUD by itself.

i see. i was avoiding to delete methods because im new to this code base

but, if you say so, ok i'll merge them

Reply:
No worries, as the methods are used only within the tests, so nobody will be affected by the change.